libraries/TGRSIAnalysis/TTigress/TTigress.cxx

```diff
@@ -130,26 +143,26 @@ Int_t TTigress::GetAddbackMultiplicity() {
     return fAddbackHits.size(); 
   }
     
-  // use the first tigress hit as starting point for the addback hits
+  // use the first (highest E) tigress hit as starting point for the addback hits
   fAddbackHits.push_back(fTigressHits[0]);
-  //I can see nothing in the current TTigressHit class or SumHit method that requires this line
-  //fAddbackHits.back().SumHit(&(fAddbackHits.back()));//this sets the last position
   fAddbackFrags.push_back(1);
 
   // loop over remaining tigress hits
   size_t i, j;
-  for(i = 1; i < fTigressHits.size(); ++i) {
+  for(i = 1; i < fTigressHits.size(); i++) {
```

Comment:
++i and ++j are the same and technically faster than i++ and j++ since the former just creates an rvalue and the right copies into a temporary lvalue